cli/integration_tests/single_package/dry-run.t

```diff
@@ -33,7 +33,7 @@ Check
   {
     "id": "[a-zA-Z0-9]+", (re)
     "version": "0",
-    "turboVersion": "[0-9\.]+", (re)
+    "turboVersion": "[a-z0-9\.]+", (re)
```

Comment:
Does this regex also accept the `-` in the version?

Reply:
blah, I should use what we already have and is working 
https://github.com/vercel/turbo/blob/a5427a7904b41e523125db3a3b7bbcddcd469c39/cli/integration_tests/basic_monorepo/dry_run_json.t#L8-L10

Reply:
https://regex101.com/r/TnLKKL/1